docs/general/server/media/Lyrics.md

```diff
@@ -0,0 +1,63 @@
+---
+uid: server-media-lyrics
+title: Lyrics
+---
+
+# Lyrics
+Lyrics are contained in the same folder and must match the filenames for their corresponding item. For example: The lyric file for `01 Death Eternal.mp3` must be `01 Death Eternal.lrc` or `01 Death Eternal.txt`.
+
+```
+Music
+└── Some Artist
+    └── Album A
+        ├── Song 1.flac
+        ├── Song 1.lrc
+        ├── Song 2.flac
+        ├── Song 2.lrc
+        ├── Song 3.flac
+        └── Song 3.lrc
+```
+
+The Lyrics file can be synchronized or unsynchronized. It can have some extra metadata, but that won't be registered by Jellyfin.
```

Comment:
>  It can have some extra metadata, but that won't be registered by Jellyfin.

Lyric metadata is actually parsed and saved, the web client just does not show it yet.